packages/eslint-config-hmpps/index.cjs

```diff
@@ -72,7 +72,7 @@ const scriptExtensionsGlob = '@(js|mjs|cjs|ts|mts|cts)'
 /**
  * Generates the HMPPS shared best-practice eslint rules for typescript projects
  *
- * @param {Object=} args
+ * @param {Object} args
```

Comment:
This was coerced to `any` when trying to do use it in vscode

Reply:
Then how do you have an actually-optional argument?